usecase/usecase-cordova-android-tests/samples/CordovaXwalkHost/README.md

```diff
@@ -0,0 +1,6 @@
+## Usecase Design
+
+This sample demonstrates cordova project app can set xwalk host, include:
+
+* '<access origin="*" />' in config of cordova project is available for setting xwalk host
```

Comment:
You mean it is default configuration? Have you try if it is failed without this tag?